tests/spdx3/bump/test_spdx_document_bump.py

```diff
@@ -13,18 +13,23 @@
 from spdx.model.document import Document as Spdx2_Document
 
 from spdx3.bump_from_spdx2.spdx_document import bump_spdx_document
-from spdx3.spdx_id_map import SpdxIdMap
-from spdx3.writer.console.spdx_id_map_writer import write_spdx_id_map
-from tests.spdx.fixtures import document_fixture
+from spdx3.payload import Payload
+from spdx3.writer.console.spdx_id_map_writer import write_payload
+from tests.spdx.fixtures import document_fixture, creation_info_fixture, annotation_fixture
 
 
 def test_bump_spdx_document():
     spdx2_document: Spdx2_Document = document_fixture()
 
-    spdx_id_map: SpdxIdMap = bump_spdx_document(spdx2_document)
+    payload: Payload = bump_spdx_document(spdx2_document)
 
-    write_spdx_id_map(spdx_id_map, sys.stdout)
+    write_payload(payload, sys.stdout)
 
-    assert "SPDXRef-Package" in list(spdx_id_map.get_full_map().keys())
+    assert "SPDXRef-Package" in list(payload.get_full_map().keys())
     assert len(
-        spdx_id_map.get_full_map().values()) == 6
+        payload.get_full_map().values()) == 6
+
+
+    #this is more of a temporary test to make sure the dates don't get messed up again
```

Comment:
Why is this temporary?

Reply:
I imagine we will do more refined testing once the dust has settled.

Reply:
Ah yes :+1: